test/controllers/project_controller_test.exs

```diff
@@ -258,17 +258,30 @@ defmodule Ask.ProjectControllerTest do
 
     test "sets archived status to true", %{conn: conn, user: user} do
       project = create_project_for_user(user)
-      put conn, project_update_archived_status_path(conn, :update_archived_status, project), project: %{"archived" => "true"}
+      put conn, project_update_archived_status_path(conn, :update_archived_status, project), project: %{"archived" => true}
```

Comment:
why this is changing from `"true"` to `true`? but on line 289 that is not done, and in questionnaire_controller_test.exs the string alternativa is also used?

Reply:
Here (and there) I just improved the code to make it test the same data type the application handles in real time.

Reply:
But the controllers handle http request query string and post body. There are no booleans there, everything is a string `project_id=2&archived=true`.

Reply:
The body sent in the HTTP request looks like `{project: {archived: true, ... }}`
The function that handles the request extracts `project_params` [here](https://github.com/instedd/surveda/blob/9af2526542c0f1d1fdd1a038c961d39141eee8fd/web/controllers/project_controller.ex#L135). It looks like `%{"archived" => true, ...}`

The test wasn't representing the real scenario where this function is being called.